datascroller/scroller.py

```diff
@@ -213,6 +214,14 @@ def query(self, query_string, viewing_area):
         #    table_name = query_words[from_index + 1]
         #    exec("%s = %d" % (table_name, self.full_df))
 
+    def get_next_chunk(self):
+        try:
+            df = self.reader.get_chunk()
+        # if we reach the last chunk, just stop there
+        except StopIteration:
```

Comment:
So, the idea is, if we hit `StopIteration`, then `full_df` is equal to the last chunk. So this function returns a new copy of the same `DFWindow`? Couldn't we just return `self`? 

More importantly, personally I think the default behavior should be to go back to the firs chunk. Probably from a UI perspective it'd be smart to print a "Reached last chunk. Returning to top" message.

Reply:
Makes sense. Let’s consider this one MVP and update behavior in subsequent PR.